dapp-oeth/src/hooks/useTokenPrices.js

```diff
@@ -0,0 +1,189 @@
+import { useEffect, useState } from 'react'
+import { useStoreState } from 'pullstate'
+import ContractStore from 'stores/ContractStore'
+import { utils } from 'ethers'
+
+const tokenConfiguration = {
+  eth: {
+    id: 'ethereum',
+    symbol: 'eth',
+    name: 'Ethereum',
+  },
+  weth: {
+    id: 'weth',
+    symbol: 'weth',
+    name: 'WETH',
+    platforms: {
+      ethereum: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+    },
+  },
+  oeth: {
+    id: 'origin-ether',
+    symbol: 'oeth',
+    name: 'Origin Ether',
+    platforms: {
+      ethereum: '0x856c4efb76c1d1ae02e20ceb03a2a6a08b0b8dc3',
+    },
+  },
+  frxeth: {
+    id: 'frax-ether',
+    symbol: 'frxeth',
+    name: 'Frax Ether',
+    platforms: {
+      ethereum: '0x5e8422345238f34275888049021821e8e08caa1f',
+    },
+  },
+  sfrxeth: {
+    id: 'staked-frax-ether',
+    symbol: 'sfrxeth',
+    name: 'Staked Frax Ether',
+    platforms: {
+      ethereum: '0xac3e018457b222d93114458476f3e3416abbe38f',
+    },
+  },
+  steth: {
+    id: 'staked-ether',
+    symbol: 'steth',
+    name: 'Lido Staked Ether',
+    platforms: {
+      ethereum: '0xae7ab96520de3a18e5e111b5eaab095312d7fe84',
+    },
+  },
+  reth: {
+    id: 'rocket-pool-eth',
+    symbol: 'reth',
+    name: 'Rocket Pool ETH',
+    platforms: {
+      ethereum: '0xae78736cd615f374d3085123a210448e74fc6393',
+    },
+  },
+}
+
+const oethOraclePrice = async (contract, tokenAddress) => {
+  try {
+    return await contract.price(tokenAddress)
+  } catch (e) {
+    console.error(e)
+    return utils.parseEther('1')
```

Comment:
Should we do this? I think I would rather have an error msg

Reply:
I'd rather the dapp still be functional than break on this error. It's just the USD value display and this shouldn't happen unless there is a bad network call

Reply:
ah ok, I thought this was from the exchange rate of the tokens